app/src/main/res/values/colors.xml

```diff
@@ -6,5 +6,10 @@
     <color name="teal_200">#FF03DAC5</color>
     <color name="teal_700">#FF018786</color>
     <color name="black">#FF000000</color>
-    <color name="white">#FFFFFFFF</color>
-</resources>
\ No newline at end of file
+    <color name="white">#FFFFFF</color>
+    <color name="grey">#E1DDDC</color>
+    <color name="red">#F30606</color>
+    <color name="as_red">#952424</color>
+    <color name="green">#FF80AB</color>
+    <color name="yellow">#D1BB5B</color>
```

Comment:
đặt lại tên color_<màu>, bỏ màu white đi android có sẵn rồi

Reply:
đặt tên màu theo đây nhé: https://chir.ag/projects/name-that-color/